GX-TimeFrequency/src/minimax_grids.F90

```diff
@@ -94,21 +131,23 @@ subroutine gx_minimax_grid(num_points, e_min, e_max, &
     ! Scale the frequency grid points and weights from [1,R] to [e_min,e_max]
     ! Note: the frequency grid points and weights include a factor of two
     scaling = e_min
-    omega_points = x_tw(1: num_points) *scaling
-    omega_weights = x_tw(num_points+1: 2* num_points) *scaling   
+    omega_points(:) = x_tw(1: num_points) *scaling
```

Comment:
`(:)` is superfluous when operating/accessing the whole array. Please remove it

Reply:
Same deal as above `-Wrealloc-lhs`

Reply:
I've had a read and in these instances, the assignment checks either shouldn't be a performance hit, or should be optimised away by the compiler.

" In hot loops, the Fortran 2003 reallocation feature may reduce the performance"

But I agree, if the compiler's still throwing out warnings then fine, make the change to remove them.